tools/export/iar/iar_definitions.json

```diff
@@ -371,7 +371,7 @@
     "STM32H743ZI": {
         "OGChipSelectEditMenu": "STM32H743ZI\tST STM32H743ZI"
     },
-    "DISCO_H747I": {
+    "STM32H747XIHx": {
```

Comment:
Is this correct? Should this be an additional section rather than a replacement?

Reply:
From my (limited) understanding of the export, yes it is necessary: without the change in iar_definitions.json, the export fails.
It is link to the fact that now there is new "device_name" in target.json for DISCO_H747I
        "device_name": "STM32H747XIHx",

Reply:
Ok, thanks for that. I guess that neither of us are experts in the IAR exporter...